docs/mounting.md

```diff
@@ -22,6 +22,10 @@ Console:
 
     GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json gcsfuse [...]
 
+On OS X and Linux you can also use fstab option in `/etc/fstab` system file:
```

Comment:
Could you touch up the wording here a bit (and drop the references to OS X and Linux, since these are the only systems we support)? Probably:

When mounting with an fstab entry, you can use the `key_file` option. For example: